tgui/packages/tgui/interfaces/Orbit/helpers.ts

```diff
@@ -66,19 +66,13 @@ const getThreatColor = (orbiters: number) => {
   }
 };
 
-/**
- * ### getDisplayColor
- * Displays color for buttons based on the health or orbiter count. Toggleable.
- * @param {Observable} item - The point of interest.
- * @param {boolean} heatMap - Whether the user has heat map toggled.
- * @param {string} color - OPTIONAL: The color to default to.
```

Comment:
I'm tuning down my level of tryhardiness and not jsdocing what is already made apparent with types